src/Layers/xrRender/HW.h

```diff
@@ -5,44 +5,31 @@
 #include "SDL.h"
 #include "SDL_syswm.h"
 
-#if !defined(_MAYA_EXPORT) && !defined(USE_OGL)
+#if !defined(_MAYA_EXPORT)
 #include "stats_manager.h"
 #endif
 
 class CHW
```

Comment:
It's better to create an interface `IHW` and derive all `CHW` classes from it.

Reply:
Sounds reasonable. I'm going to move existing `CHW` (DX9 device) into `xrRenderDX9/dx9HW.h|cpp` and use `HW.h` for interface declaration.
Is it ok?

Reply:
UPD: accesses to members like `pDevice` are scattered in the code. Introducing the interface for `CHW` will cause to adding lots of explicit downcasts and won't solve anything.
Let's just move `CHW.cpp` and `CHWCaps.cpp` from common render folder into DX9-specific implementation

Reply:
> will cause to adding lots of explicit downcasts

Why it would to? We don't need to add any members to interface; interface is only about functions.

But... 
> Let's just move CHW.cpp and CHWCaps.cpp from common render folder into DX9-specific implementation

Sounds enough. Probably, interface would be excessive, so...